fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -244,9 +247,21 @@
         managedProperties = b.build();
     }
 
+    public static final Set<Property> relaxableProperties
+            = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(new Property[]{
```

Comment:
If you have stylistic reasons for the syntax here, that is fine. If not, however, the following seems more readable:
```
    public static final Set<Property> relaxableProperties
            = of(LAST_MODIFIED_BY, LAST_MODIFIED_DATE, CREATED_BY, CREATED_DATE);
```